webapp/package.json

```diff
@@ -10,10 +10,17 @@
   "scripts": {
     "dev": "yarn workspaces foreach -pi run dev",
     "build": "yarn workspaces foreach -pt run build",
+    "postbuild": "cp -r client/build server/build/client",
```

Comment:
`post*` have been removed and will not auto executed after `build`. Please find another way to do so.